README.md

```diff
@@ -21,20 +24,17 @@ Unit testing of the application is performed within
 [yarn](https://yarnpkg.com/) is used to manage the build/test/deploy cycle.
 
 #### Screenshot
-
-![screen August 17, 2017](documentation/screenshot.png)
+![Consumer Financial Protection Bureau - Google Chrome 20-10-2023 10_37_09 PM](https://github.com/kiranrokkam09/ccdb5-ui/assets/85286397/90b9e381-f960-4c49-b956-0b8a254bcae3)
 
 ## Dependencies
 
-This application depends on the following third-party components:
+This application leverages a robust design system, which serves as the foundation for maintaining a consistent and cohesive user interface. While the design system relies on various components and libraries, we'll focus on the design system as a whole, eliminating the need to list each individual dependency.
 
-1. [Capital Framework](https://cfpb.github.io/capital-framework/) - CFPB standard styling and controls
-1. [History](https://github.com/reacttraining/history) - Integrating the address bar with the application
-1. [dayjs](https://day.js.org/) - Better date handling than native JavaScript
+For detailed information about the design system and its underlying components, please refer to the official [Design System Documentation](https://thedesignsystem.guide/documentation). The design system provides standardized styling, controls, and guidelines, ensuring a unified user experience throughout the application.
```

Comment:
This is not the design system we use. The CFPB has its own design system:
https://cfpb.github.io/design-system/

Reply:
Also, I think I'd dial back the verbiage on the design system. We don't need to sell it.

Reply:
ok then i will make the suggested changes.